engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2405,6 +2540,46 @@ private LogicalOperatorInfo fillLogicalOperatorInfo(OperatorMeta operator)
     return loi;
   }
 
+  private LogicalModuleInfo fillLogicalModuleInfo(ModuleMeta module, boolean flatten, String parentModuleName)
+  {
+    LogicalModuleInfo lmi = new LogicalModuleInfo();
+    lmi.name = module.getName();
+    lmi.className = module.getModule().getClass().getName();
+    if (flatten) {
+      for (OperatorMeta operatorMeta : getLogicalPlan().getAllOperators()) {
```

Comment:
Will module.getDag().getAllOperators() not return the same thing i.e. information of operators which were populated by this module??